pkg/porter/outputs.go

```diff
@@ -2,14 +2,31 @@ package porter
 
 import (
 	"encoding/json"
+	"fmt"
 	"os"
 	"path/filepath"
 	"sort"
 	"strings"
 
+	"github.com/deislabs/porter/pkg/context"
+	"github.com/deislabs/porter/pkg/printer"
+	tablewriter "github.com/olekukonko/tablewriter"
 	"github.com/pkg/errors"
 )
 
+// OutputShowOptions represent options for a bundle output show command
+type OutputShowOptions struct {
+	sharedOptions
+	Output string
+}
+
+// OutputListOptions represent options for a bundle output list command
+type OutputListOptions struct {
+	sharedOptions
+	RawFormat string
```

Comment:
This is a question about something we could look at in a follow-up, not in this PR since I think it may touch a couple commands. 

Do we have these (raw format and format) in a common struct? Would it help if we did or would it just make another struct but not help consolidate validation logic?

Reply:
I don't think we do yet... but I wonder if we just add the `RawFormat` field to https://github.com/deislabs/porter/blob/4d5fbb8636186575bc856a98997929f8bce17b00/pkg/printer/printer.go#L24-L26 and we'd have what we need to remove duplication?

Reply:
I like it! Yeah that would be a nice to have at some later point, but not this PR. I have an itchy trigger finger. 😀